ArchUnitNET/Fluent/EvaluationResult.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ArchUnitNET.Domain;
+using JetBrains.Annotations;
+
+namespace ArchUnitNET.Fluent
+{
+    public class EvaluationResult
+    {
+        public EvaluationResult([CanBeNull] ICanBeAnalyzed obj, bool passed, string description,
+            ICanBeEvaluated archRule, Architecture architecture)
+        {
+            Object = obj;
+            Passed = passed;
+            Description = description;
+            ArchRule = archRule;
+            Architecture = architecture;
+        }
+
+        public ICanBeEvaluated ArchRule { get; }
+        [CanBeNull] public ICanBeAnalyzed Object { get; }
```

Comment:
Object is too abstract, a more specific name will be nice

Reply:
renamed to `EvaluatedObject`